custom/icds_reports/ucr/data_sources/dashboard/delivery_forms.json

```diff
@@ -35,15 +35,13 @@
         "expression": {
```

Comment:
the `nested` expression isn't needed here since you've added the `location_property` field. You can replace the entire expression value with the value of  `argument_expression`.